mongo/mongo.go

```diff
@@ -0,0 +1,362 @@
+//go:build !nomongo
+// +build !nomongo
+
+package mongo
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
We prefer using the stdlib "errors" package. Can you remove usages of this, and use `fmt.Errorf("... %w", err)` to wrap errors?